src/jenkins_job_insight/main.py

```
@@ -1051,12 +1202,179 @@ async def dashboard(
+    logger.debug(
+        f"GET /history/trends: period={period}, days={days}, job_name={job_name!r}"
+    )
+    return await storage.get_trends(
+        period=period, days=days, job_name=job_name, exclude_job_id=exclude_job_id
+    )
+
+
+@app.post("/history/classify", status_code=201)
+async def classify_test(request: Request, body: dict) -> dict:
+    """Classify a test as FLAKY, REGRESSION, etc. Used by AI and humans."""
+    logger.debug(f"POST /history/classify: body_keys={list(body.keys())}")
```

Ruff (0.15.6)

[warning] Logging statement uses f-string

(G004)